src/engine/client/cl_main.cpp

```diff
@@ -926,15 +738,7 @@ void CL_ShutdownAll()
 	cls.soundRegistered = false;
 
 	// Gordon: stop recording on map change etc, demos aren't valid over map changes anyway
-	if ( clc.demorecording )
-	{
-		CL_StopRecord_f();
-	}
-
-	if ( clc.waverecording )
-	{
-		CL_WavStopRecord_f();
-	}
+    CL_StopRecord();
```

Comment:
indentation looks wrong... I know we are fighting competing styles, but we should at least match the code that we are around.